### PR TITLE
doc: document mds journal event types

### PR DESCRIPTION
**Pull request by vshankar**

### CODE REVIEW

**Comment by vshankar** [comment]
jenkins render doc

**Comment by vshankar** [comment]
jenkins render docs

File: doc/cephfs/mds-journaling.rst [hunk 1]
```diff
@@ -55,3 +55,36 @@ line utility `cephfs-journal-tool` which can be used as follows:
 
 `cephfs-journal-tool` is also used to discover and repair a damaged Ceph File System.
 (See :doc:`/cephfs/cephfs-journal-tool` for more details)
+
+Journal Event Types
```
Comment (fullerdj) [inline doc/cephfs/mds-journaling.rst]: What do you think about making a reference to the header file here?

Comment (vshankar) [reply doc/cephfs/mds-journaling.rst]: My concern with that is that it may lead to a developer inclined documentation rather than for the wider audience.

**Pull request merged**